build failed on Raspberry Pi

**V version:** 
0.1.8
**OS:** 
Raspbian GNU/Linux 8
```
$ gcc -v
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/arm-linux-gnueabihf/4.9/lto-wrapper
Target: arm-linux-gnueabihf
Configured with: ../src/configure -v --with-pkgversion='Raspbian 4.9.2-10+deb8u2' --with-bugurl=file:///usr/share/doc/gcc-4.9/README.Bugs --enable-languages=c,c++,java,go,d,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.9 --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.9 --libdir=/usr/lib --enable-nls --with-sysroot=/ --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --enable-gnu-unique-object --disable-libitm --disable-libquadmath --enable-plugin --with-system-zlib --disable-browser-plugin --enable-java-awt=gtk --enable-gtk-cairo --with-java-home=/usr/lib/jvm/java-1.5.0-gcj-4.9-armhf/jre --enable-java-home --with-jvm-root-dir=/usr/lib/jvm/java-1.5.0-gcj-4.9-armhf --with-jvm-jar-dir=/usr/lib/jvm-exports/java-1.5.0-gcj-4.9-armhf --with-arch-directory=arm --with-ecj-jar=/usr/share/java/eclipse-ecj.jar --enable-objc-gc --enable-multiarch --disable-sjlj-exceptions --with-arch=armv6 --with-fpu=vfp --with-float=hard --enable-checking=release --build=arm-linux-gnueabihf --host=arm-linux-gnueabihf --target=arm-linux-gnueabihf
Thread model: posix
gcc version 4.9.2 (Raspbian 4.9.2-10+deb8u2)
```
**What did you do?**
make

**What did you expect to see?**
build success
 
**What did you see instead?**
```
$ make
rm -f v.c .v.c v vprod thirdparty/**/*.o
find . -name '.*.c' -print0 | xargs -0 -n1 rm -f
curl -Os https://raw.githubusercontent.com/vlang/vc/master/v.c
cc -std=gnu11 -w -o v v.c -lm
./v -o v compiler
.v.c: In function ‘array_repeat’:
.v.c:1260:2: error: ‘for’ loop initial declarations are only allowed in C99 or C11 mode
  for (
  ^
.v.c:1260:2: note: use option -std=c99, -std=gnu99, -std=c11 or -s...
(Use `v -debug` to print the entire error message)

V panic: C error. This should never happen. Please create a GitHub issue: https://github.com/vlang/v/issues/new/choose
Makefile:9: recipe for target 'v' failed
make: *** [v] Error 1
```
**Some Debug**

```
$ ./v -show_c_cmd -o v compiler | head

==========
cc  -w -g -o v ".v.c"   -lm -lpthread   -ldl
cc took 7226 ms
=========

.v.c: In function ‘array_repeat’:
.v.c:1260:2: error: ‘for’ loop initial declarations are only allowed in C99 or C11 mode
  for (
  ^
```

build success after add `-std=gnu11`

```
$ cc -std=gnu11  -w -g -o v ".v.c"   -lm -lpthread   -ldl
$
```

--- a/compiler/main.v
+++ b/compiler/main.v
@@ -710,7 +710,7 @@ fn (v mut V) cc() {
 	} 
 	linux_host := os.user_os() == 'linux'
 	v.log('cc() isprod=$v.pref.is_prod outname=$v.out_name')
-	mut a := [v.pref.cflags, '-w'] // arguments for the C compiler
+	mut a := [v.pref.cflags, '-std=gnu11', '-w'] // arguments for the C compiler
 	flags := v.table.flags.join(' ')
 	//mut shared := ''
 	if v.pref.is_so {

